how-to-team-desktop/issues.md

```diff
@@ -0,0 +1,85 @@
+
+**Definitions**
+
+| Word | Definition |
+|------|------------|
+| Duplicate | An open or closed issue already exists for the issue. |
+| Reproducible | The issue can be encountered by someone other than the opener. |
+| Stale | No activity has been received on an issue/PR within a set amount of time. Recommended time to be considered stale is 60 days, but can be variable depending on the project and issue. Generally should be automated. |
+
+## Labels
+
+*Edit repository labels at `https://github.com/<organization>/<repository>/labels`.*
+
+Individual repositories should use these labels in addition to repository-specific labels to further classify issues and pull requests. For example, the [Steam Snap](https://github.com/canonical/steam-snap/labels) has labels to classify an issue further such as `game-specific`, `client-specific`, `hardware-specific`, etc that are labels exclusive to the Snap. Many repositories may not need additional labels other than the ones defined below.
+
+### Types
+
+Generally an issue or PR should have exactly one of the following labels indiciating the type of the issue or PR. These may be automatically applied by using [issues templates](#templates) and *usually are never changed or removed once added*.
+
+| Label | Use Case |
+|-------|----------|
+| `type/bug` | Issues that are related to a problem or unexpected behavior. |
+| `type/documentation` | Issues or PRs relating to documentation or wiki pages. |
+| `type/enhancement` | Issues or PRs relating to new feature requests. |
+| `type/question` | Issues requesting information or to have a discussion. |
+
+### Triage States
+
+Labels to be applied while triaging an issue, indicating the result of triage. Usually these states lead to the issue or PR being closed once applied.
+
+| Label | Use Case |
+|-------|----------|
+| `triage/duplicate` | Issues or PRs that already have an existing issue or PR. |
+| `triage/invalid` | Issues or PRs that are not relevant. |
+| `triage/not-reproducible` | Issues that cannot be encountered by someone else. |
+| `triage/stale` | Issues or PRs that have not had activity in a set amount of time, usually ~60 days. |
+| `triage/upstream` | Issues that are actually an upstream issue, not an issue with this repository. |
+
+### Priority
+
+Labels to be applied indicating the priority of an issue or PR.
+
+| Label | Use Case |
+|-------|----------|
+| `priority/high` | Issues or PRs with high priority. |
+| `priority/medium` | Issues or PRs with medium priority. |
+| `priority/low` | Issues or PRs with low priority. |
+
+### Further Labels
+
+Labels that may be added or removed at any point to an issue or PR that further documents its state.
+
+| Label | Use Case |
+|-------|----------|
+| `good-first-issue` | Issues that are good for first-time contributors. These should be small and well defined/documented fixes or features. |
+| `help-wanted` | Issues or PRs that may need additional help from others. |
```

Comment:
I think we can make this description narrower. To me `good-first-issue` is a subset of `help-wanted` where the barrier to entry is lowest, making them good project entry points. Both are:

- Lower priority issues
- Clear boundaries for an external contributor to know what the acceptance criteria for a pr would be
- Useful enough contribution that it is worth the time on a maintainer to invest into a code review if a pr is made

I particularly like the explanation given [here](https://www.kubernetes.dev/docs/guide/help-wanted/) of these tags.

Reply:
Updated in https://github.com/canonical/desktop-engineering/pull/16/commits/47375e94a985638db3e1b11f7ddfde910f84a592, let me know if that's what you were thinking.